src/script/page/MainContent/panels/preferences/accountPreferences/AccountInput.tsx

```diff
@@ -99,7 +101,7 @@ const AccountInput: React.FC<AccountInputProps> = ({
     return () => {
       document.removeEventListener('mousedown', handleClickOutside);
     };
-  }, [textInputRef]);
+  }, [inputWrapperRef]);
```

Comment:
This will never trigger since it's a `ref`, react doesn't listen to `ref` changed

Reply:
well, to be fair, the previous code should not work as well.. 🙃